js/views/userPageVw.js

```diff
@@ -505,7 +504,7 @@ module.exports = Backbone.View.extend({
 
     $.ajax({
       type: "DELETE",
-      url: self.item.get('server') + "contracts/?id="+ self.item.get('id'),
+      url: self.item.get('server') + "contracts/?id="+ self.item.get('id'), //TODO: Change from GET to POST for safe methods, http://restcookbook.com/HTTP%20Methods/idempotency/
```

Comment:
See the table on the link provided, and see how DELETE is not a safe method, and should have the arguments in the POST body.